tests/test_cfg_end_to_end_tests.py

```diff
@@ -157,7 +159,8 @@ def test_configs(config_path: str, data_check_fn: Callable, tmp_path: str):
     assert "processors" in cfg
     cfg["processors_to_run"] = "all"
     cfg["workspace_dir"] = str(tmp_path)
-    cfg["final_manifest"] = str(tmp_path / "final_manifest.json")
+    if "final_manifest" not in cfg:
```

Comment:
Why is this needed? I think it's important to override it no matter what, otherwise we will not be able to find it locally

Reply:
Since we have final_manifest key in the config file it can lead to misunderstanding. User can define it as "my_final_manifest.json" but we are looking for "final_manifest.json" no matter what is defined.